### ISSUE TITLE
Typo in readme.md

### ISSUE THREAD

**Issue by richard-scott** [cogitatio/vagrant-hostsupdater#100]
There is a typo [here](https://github.com/cogitatio/vagrant-hostsupdater/blob/master/README.md#keeping-host-entries-after-suspendhalt):

```
config.hostsupdater.resume_on_suspend = false
```

Should be this:

```
config.hostsupdater.remove_on_suspend = false
```

Otherwise Vagrant errors on startup:

```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

VagrantPlugins::HostsUpdater::Config:
* The following settings shouldn't exist: resume_on_suspend
$
```

**Comment by cgsmith** [comment]
Thanks @richard-scott 

This was fixed in master here 7ee3be1 and will be released in 1.0.3

**Comment by richard-scott** [comment]
Thanks :-)
